backend/public/index.html

```diff
@@ -0,0 +1 @@
+HAALO!
```

Comment:
Do we need this file? (maybe it's required.. who knows 🤷) but maybe then different file contents?

Reply:
added option `index: false`, otherwise it looks for a default index.html page to returns, otherwise it throws. removed index.html.

Reply:
haah love this